app/javascript/packs/reducers/questions.js

```diff
@@ -120,128 +128,44 @@ function optionFetchSuccess(state, action) {
     ...state,
     [questionId]: {
       ...state[questionId],
-      fetchStatus: action.fetchStatus,
       options: {
         ...state[questionId].options,
         [optionId]:{
-          ...action.response
+          ...action.response,
+          fetchStatus: action.fetchStatus,
         }
       }
     }
   }
 }
 
 function optionFetchFailure(state, action) {
-  return state
-}
-
-function beforeCreateOption(state, action) {
   const questionId = action.option.question_id;
   const optionId = action.option.id;
   return {
     ...state,
     [questionId]: {
       ...state[questionId],
-      fetchStatus: action.fetchStatus,
       options: {
         ...state[questionId].options,
         [optionId]: {
           ...action.option,
-          temp: true
-        }
-      }
-    }
-  }
-}
-
-function beforeFetchQuestion(state, action) {
-  const questionId = action.question.id;
-  return {
-    ...state,
-    [questionId]: {
-      ...state[questionId],
-      saved: false
-    }
-  }
-}
-
-function beforeFetchOption(state, action) {
-  const questionId = action.option.question_id;
-  const optionId = action.option.id;
-  return {
-    ...state,
-    [questionId]: {
-      ...state[questionId],
-      fetching: action.fetching,
-      saved: false,
-      options: {
-        ...state[questionId].options,
-        [optionId]: action.option
-      }
-    }
-  }
-}
-
-function beforeCreateQuestion(state, action) {
-  const questionId = action.question.id;
-  return {
-    ...state,
-    [questionId]: {
-      ...action.question,
-      saved: false,
-      temp: true
-    }
-  }
-}
-
-function questionFetchSuccess(state, action) {
```

Comment:
Sorry these are the duplicates that got in during my merge with Helen's branch -- I pushed a change to remove them, and the actual functions (with fetchStatus instead of saved/fetching) are still in the file